core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/inheritance/singlevarinterface/baseannotated/childannotated/TestConstraintProvider.java

```diff
@@ -0,0 +1,22 @@
+package ai.timefold.solver.core.impl.testdata.domain.inheritance.singlevarinterface.baseannotated.childannotated;
```

Comment:
Example of shortening: 
```suggestion
package ai.timefold.solver.core.impl.testdata.domain.inheritance.singlevarinterface.baseannotated.childnot;
```